ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -171,20 +171,25 @@ SVFrequecyHeaderFilter.propTypes = {
   onChange: PropTypes.func,
 }
 
-const QS_FILTER_FIELD = {
+const SV_QS_FILTER_FIELD = {
   name: 'min_qs',
-  label: 'SV Quality Score',
-  labelHelp: (
-    <span>The quality score (QS) represents the quality of a Structural Variant call. Recommended SV-QS cutoffs for filtering:<br />
-      WGS: &gt; 10; <br />
-      WES: duplication &gt;= 50, deletion &gt;= 100, homozygous deletion &gt;= 400.
-    </span>),
+  label: 'Exome SV Quality Score',
+  labelHelp: 'The quality score (QS) represents the quality of a Structural Variant call. Recommended SV-QS cutoffs for filtering: duplication &gt;= 50, deletion &gt;= 100, homozygous deletion &gt;= 400.',
```

Comment:
this is a string not HTML so you should use string literals and not html symbols